ores/applications/celery.py

```diff
@@ -0,0 +1,58 @@
+"""
+Runs a celery worker.
+
+Usage:
+    celery [--config-dir=<path>]... [--logging-config=<path>]
+
+Options:
+    -h --help                Prints this documentation
+    --config-dir=<path>      The path to a directory containing configuration
+                             [default: config/]
+    --logging-config=<path>  The path to a logging configuration file
+"""
+import glob
+import logging
+import logging.config
+import os.path
+
+import docopt
+import yamlconf
+
+from ..score_processors import Celery
+
+DEFAULT_DIRS = ["config/", "/etc/ores/"]
+
+
+def main(argv=None):
+    args = docopt.docopt(__doc__, argv=argv)
+
+    run(config_dirs=args['--config-dir'],
+        logging_config=args['--logging-config'])
+
+
+def run(*args, **kwargs):
+    application = build(*args, **kwargs)
```

Comment:
If that's the case, then I would suggest naming the methods such that their intent is clear.

> Generally, it's best to pass args, *kwargs rather than explicit arguments.
Source? This is the opposite of what I've encountered elsewhere.